gravitee-apim-console-webui/src/services-ngx/endpoint.service.ts

```diff
@@ -17,25 +17,52 @@ import { HttpClient } from '@angular/common/http';
 import { Inject, Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
 import { GioJsonSchema } from '@gravitee/ui-particles-angular';
+import { MatIconRegistry } from '@angular/material/icon';
+import { DomSanitizer } from '@angular/platform-browser';
+
+import { BASE_64_PREFIX } from './entrypoint.service';
 
 import { ConnectorListItem } from '../entities/connector/connector-list-item';
 import { Constants } from '../entities/Constants';
+import { PluginMoreInformation } from '../entities/plugin/PluginMoreInformation';
 
 @Injectable({
   providedIn: 'root',
 })
 export class EndpointService {
-  constructor(private readonly http: HttpClient, @Inject('Constants') private readonly constants: Constants) {}
+  constructor(
+    private readonly http: HttpClient,
+    @Inject('Constants') private readonly constants: Constants,
+    private readonly matIconRegistry: MatIconRegistry,
+    private _sanitizer: DomSanitizer,
+  ) {}
 
   v4ListEndpointPlugins(): Observable<ConnectorListItem[]> {
-    return this.http.get<ConnectorListItem[]>(`${this.constants.env.baseURL}/v4/endpoints`);
+    return this.http.get<ConnectorListItem[]>(`${this.constants.baseURL}/v4/endpoints`);
   }
 
   v4Get(id: string): Observable<ConnectorListItem> {
     return this.http.get<ConnectorListItem>(`${this.constants.env.baseURL}/v4/endpoints/${id}`);
   }
 
   v4GetSchema(id: string): Observable<GioJsonSchema> {
-    return this.http.get<GioJsonSchema>(`${this.constants.env.baseURL}/v4/endpoints/${id}/schema`);
+    return this.http.get<GioJsonSchema>(`${this.constants.baseURL}/v4/endpoints/${id}/schema`);
+  }
+
+  v4GetMoreInformation(endpointId: string): Observable<PluginMoreInformation> {
+    return this.http.get<PluginMoreInformation>(`${this.constants.baseURL}/v4/endpoints/${endpointId}/moreInformation`);
+  }
+
+  registerSvg(endpoint): string {
```

Comment:
WDYT to move this in a shared utils service ? like icon.util.ts ?

Reply:
Yes, Goog idea